### PR TITLE
feat: support speech synthesis model

### PR DESCRIPTION
**Pull request by kevinlin09**
feat: support speech synthesis model

support both Sambert and CosyVoice

### CODE REVIEW

File: spring-ai-alibaba-core/src/main/java/com/alibaba/cloud/ai/dashscope/api/DashScopeSpeechSynthesisApi.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
+package com.alibaba.cloud.ai.dashscope.api;
```
Comment (yuluo-yx) [inline spring-ai-alibaba-core/src/main/java/com/alibaba/cloud/ai/dashscope/api/DashScopeSpeechSynthesisApi.java:1]: pls add license header

File: spring-ai-alibaba-core/src/main/java/com/alibaba/cloud/ai/dashscope/audio/synthesis/DashScopeSpeechSynthesisModel.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
+package com.alibaba.cloud.ai.dashscope.audio.synthesis;
```
Comment (yuluo-yx) [inline spring-ai-alibaba-core/src/main/java/com/alibaba/cloud/ai/dashscope/audio/synthesis/DashScopeSpeechSynthesisModel.java:1]: ditto

File: spring-ai-alibaba-core/src/main/java/com/alibaba/cloud/ai/dashscope/audio/synthesis/SpeechSynthesisModel.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+ * @author kevinlin09
+ */
+public interface SpeechSynthesisModel extends Model<SpeechSynthesisPrompt, SpeechSynthesisResponse> {
+
+	@Override
+	SpeechSynthesisResponse call(SpeechSynthesisPrompt prompt);
+
+	Flux<SpeechSynthesisResponse> stream(SpeechSynthesisPrompt prompt);
+
+}
```
Comment (yuluo-yx) [inline spring-ai-alibaba-core/src/main/java/com/alibaba/cloud/ai/dashscope/audio/synthesis/SpeechSynthesisModel.java:17]: add new line

File: spring-ai-alibaba-core/src/main/java/com/alibaba/cloud/ai/dashscope/protocol/DashScopeWebSocketClient.java [hunk 4]
```diff
@@ -0,0 +1,277 @@
+package com.alibaba.cloud.ai.dashscope.protocol;
+
+import com.alibaba.cloud.ai.dashscope.api.ApiUtils;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import okhttp3.*;
```
Comment (yuluo-yx) [inline spring-ai-alibaba-core/src/main/java/com/alibaba/cloud/ai/dashscope/protocol/DashScopeWebSocketClient.java:8]: Should we avoid full imports?

File: spring-ai-alibaba-core/src/main/java/com/alibaba/cloud/ai/dashscope/api/DashScopeSpeechSynthesisApi.java (hunk 1 above)
Comment (kevinlin09) [reply spring-ai-alibaba-core/src/main/java/com/alibaba/cloud/ai/dashscope/api/DashScopeSpeechSynthesisApi.java:1]: done

**Pull request merged**